toplevel/topcommon.ml

```diff
@@ -274,7 +274,7 @@ let set_paths () =
       [expand "+camlp4"];
     ]
   in
-  Load_path.init ~auto_include:Compmisc.auto_include load_path;
+  Load_path.init ~auto_include load_path;
```

Comment:
Not strictly related, but don't we want to remove the `[expand "+camlp4"]` line above, which has been a no-op for years?